src/css/app.scss

```diff
@@ -266,10 +278,32 @@ footer {
 
                 input[type="text"] {
                     flex: 1 0;
-                    padding-left: 24px;
-                    color: #{$dark-grey};
+                    margin-right: 24px;
+                    color: $black;
                     font-size: 1.3em;
 
+                    border: 0;
+                    border-radius: 0;
+                    border-bottom: 1px solid $black;
+                    background-color: transparent;
+                    box-shadow: none;
+                    transition: all 0.5s ease-in;
+
+                    &:focus {
+                        outline: 0;
+                        box-shadow: none;
+                        border-color: $black;
+                    }
+
+                    &:-webkit-autofill,
+                    &:-webkit-autofill:hover,
+                    &:-webkit-autofill:focus,
+                    &:-webkit-autofill:active,
+                    &:-internal-autofill-selected,
+                    &:-internal-autofill-previewed  {
+                        -webkit-box-shadow: 0 0 0 30px white inset !important;
```

Comment:
This is a fun one, what's the goal of these guys?

Reply:
It's to change the background color of the input on autofill, so it's not a funky browser default color.